src/components/ActivityForm/ActivityForm.test.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
+import { daysOfTheWeek } from 'constants/dates';
+import testRenderErrorOnBlur from 'helpers/tests';
+import inputTypesArray from 'tests/fixtures/inputTypesArray';
+import testState from 'tests/fixtures/state';
 import ActivityForm from './ActivityForm';
```

Comment:
let's keep with the convention:
`packages`
empty line
`absolute imports`
emtpy line
`relative imports`